### PR TITLE
Change CI to Github actions and fix tests

### PR DESCRIPTION
**Pull request by querti**

### CODE REVIEW

**Comment by querti** [comment]
Tests were broken due to API changes in iiblib. Github actions don't seem to be triggered until they are pushed to master, here is the result: https://github.com/querti/pubtools-iib/actions/runs/406007292 . Tests for Python 2.6 could not be added because the dependency 'pubtools-pulplib' is not compatible with Python 2.6.

**Comment by rohanpm** [comment]
> Tests for Python 2.6 could not be added because the dependency 'pubtools-pulplib' is not compatible with Python 2.6.

That's not true, pubtools-pulplib is compatible with Python 2.6.

**Comment by querti** [comment]
> > Tests for Python 2.6 could not be added because the dependency 'pubtools-pulplib' is not compatible with Python 2.6.
> 
> That's not true, pubtools-pulplib is compatible with Python 2.6.

@rohanpm I ran into some trouble with the attrs library. Pubtools-pulplib uses the 'evolve' function, which was added to attrs after it became incompatible with Python 2.6. Evolve was added in 17.1.0 while the library dropped its Python 2.6 compatibility in 16.0.0. As a result, I am unable to install a version which has all the necessary functions.

**Comment by rohanpm** [comment]
> > > Tests for Python 2.6 could not be added because the dependency 'pubtools-pulplib' is not compatible with Python 2.6.
> > 
> > 
> > That's not true, pubtools-pulplib is compatible with Python 2.6.
> 
> @rohanpm I ran into some trouble with the attrs library. Pubtools-pulplib uses the 'evolve' function, which was added to attrs after it became incompatible with Python 2.6. Evolve was added in 17.1.0 while the library dropped its Python 2.6 compatibility in 16.0.0. As a result, I am unable to install a version which has all the necessary functions.

Ah, I see.  Yeah, that has been worked around at the RPM level for RHEL6 packages, because the relevant attrs code is actually compatible with Python 2.6, it's just the egg metadata which makes it uninstallable.  Not sure if there is any easy solution to this with just using tox/pip (and doubt it's worth to pursue anyway).

**Review by lipoja — APPROVED** [review]

File: tests/utils.py [hunk 1]
```diff
@@ -30,19 +30,24 @@ def setup_task(
             "state_reason": "state_reason",
             "state_history": [],
             "from_index": index,
+            "bundles": map_or_op,
```
Comment (midnightercz) [inline tests/utils.py:33]: why are these changes meddled with this commit?

**Review by midnightercz — CHANGES_REQUESTED** [review]

File: tests/utils.py (hunk 1 above)
Comment (querti) [reply tests/utils.py:33]: Tests needed to be fixed due to iiblib changes

**Comment by querti** [comment]
@release-engineering/distribution pls review

**Comment by querti** [comment]
Abandoning as Travis CI functionality has been restored

**Pull request closed without merging**